pyro/optim/pytorch_optimizers.py

```diff
@@ -0,0 +1,21 @@
+import torch
```

Comment:
same comment as above. for the user, is there a purpose for distinguishing between pyro wrapped pt optimizers and custom ones

Reply:
Not really. I suppose both of these should be available anyhow through the wrapping below, so we can just remove these.

Reply:
One reason is that the below snippet correctly generates links to PyTorch docs and will not correctly generate links to Pyro docs. It would take additional logic to be able to generate links for both libraries, including work-arounds for PyTorch's convention of locating docs elsewhere than `type(x).__module__`.

Reply:
@fritzo - Our wrappers for `ClippedAdam` and `AdagradRMSProp` are exactly the same as the other wrappers below, and the docstring isn't really doing much in the way of explanation. Just removing these and linking to the original PyTorch implementations like below should be self-explanatory, I think. Let me know.

Reply:
That's fine, just please preserve the links to upstream docs, as I find them useful

Reply:
@jpchen - Sorry, I was mistaken to think that the two custom Pyro optimizers were basically wrapping over PyTorch. I think a lot of our code in general is organized in a way to facilitate doc generation; so changing that seems like a bigger hack than it is worth at this point. It is also the case with distributions where the PyTorch distributions are distinguished from Pyro ones like `Delta`. We can revisit this organization if it doesn't make too much sense for documentation purposes, but we should do that in a separate issue and PR.